addons/event/views/event_slot_views.xml

```diff
@@ -8,11 +8,15 @@
             <form>
                 <group>
                     <group>
-                        <field name="start_hour" string="From" widget="float_time"/>
-                        <field name="color" widget="color_picker"/>
+                        <label for="start_hour" string="Hour range"/>
+                        <div>
+                            <field name="start_hour" class="oe_inline o_input_5ch" widget="float_time"/>
+                            <i class="fa fa-arrow-right mx-2" title="to"/>
+                            <field name="end_hour" class="oe_inline o_input_5ch" widget="float_time"/>
+                        </div>
                     </group>
                     <group>
-                        <field name="end_hour" string="To" widget="float_time"/>
+                        <field name="color" widget="color_picker"/>
                     </group>
                 </group>
             </form>
```

Comment:
Some top padding is missing here, there is overlapping content

Reply:
Actually it is there: 

<img width="651" height="357" alt="image" src="https://github.com/user-attachments/assets/8f814adc-5615-42f9-acca-e7f2c4e9420e" />

Reply:
Created a new commit for this as it lacked some padding between the picker and the actual form :smile: